src/main/java/seedu/student/ui/HelpWindow.java

```diff
@@ -18,23 +18,17 @@
     public static final String DIVIDER = "\n -------------------------------------------------------------------------"
             + "----------------------------------------------------";
 
-    public static final String COMMAND_SUMMARY = " Here is a list of all commands used for VAX@NUS" + DIVIDER
+    public static final String COMMAND_SUMMARY = " Here is a list of commonly used commands used for VAX@NUS" + DIVIDER
```

Comment:
I think there is some repetition of the word 'used' here. Maybe it could be 'Here is a list of common commands used in Vax@NUS" or "Here is a list of commonly used commands in Vax@NUS"?

Also sorry, I think we should standardize that the capitalization as Vax@NUS.

Reply:
Thanks for pointing it out! Yup, will make changes to it!